src/posix/platform/netif.cpp

```diff
@@ -1213,6 +1246,12 @@ static void processNetifAddrEvent(otInstance *aInstance, struct nlmsghdr *aNetli
             addr6.sin6_family = AF_INET6;
             memcpy(&addr6.sin6_addr, RTA_DATA(rta), sizeof(addr6.sin6_addr));
 
+            // Ignore the required anycast address
+            if (isRequiredAnycast(addr.GetBytes(), ifaddr->ifa_prefixlen))
+            {
+                ExitNow();
```

Comment:
Is this logic and `ExitNow()` correct and what we want? 

We are iterating over thr `rta` entries. As soon as we see a `RequiredAnycast()` we exit and not go throiugh any other addresses. Do we want `ExitNow()` or `continue`?

Reply:
Thanks! Updated!